.github/workflows/coverage.yml

```diff
@@ -82,3 +82,4 @@ jobs:
           files: ./build/coverage.info
           fail_ci_if_error: true
           verbose: true
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
TOKEN are mandatory only for private repository, because codecov is not able to connect and read the project to get informations.

But, when you don't give token to codecov, it use a github's API call to get information (otherwise no).

codecov got limit of call/minutes. The bug we saw last days, where about codevoc have reach this limit. So, like that there is less random bug.